taxonomy-manager-rest-server/src/main/resources/db/migration/V9__soft_delete.sql

```diff
@@ -0,0 +1,68 @@
+-- CONCEPT
+
+alter table skos_concept
+    add deleted boolean default false not null;
+
+create index skos_concept_note_deleted
+    on skos_concept (deleted)
```

Comment:
Hm, not sure what this index is for. We create an index over the 2 possible values of `deleted`: [0,1]. If we filter out the `deleted` records, this index only stores records that aren't deleted. Since `deleted` is the only key though, there's no range scans or quick lookups we get from this.

Reply:
Yeah, now that I think about it, it would probably be better to check what is used to query (id, uuid, source) and create partial indexes.

Reply:
I did it and reverted from it for some reason, brain f...og.